### PR TITLE
[MIG][15.0]mail_activity_board

### PR DESCRIPTION
**Pull request by JoanMForgeFlow**
Standard migration to v15
@AaronHForgeFlow 
@bernatpforgeflow
@mariadforgeflow

### CODE REVIEW

**Comment by AaronHForgeFlow** [comment]
@OCA/social-maintainers  can someone please approve the workflow to run here? Thanks in advance.

**Review by MiquelRForgeFlow — APPROVED** [review]

**Review by AaronHForgeFlow — CHANGES_REQUESTED** [review]
Got an error when installing contacts and creating an activity in a partner. That is not happening if I uninstall the module:

File: mail_activity_board/static/src/components/chatter_topbar/chatter_topbar.js [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+patch(
+    components.ChatterTopbar.prototype,  "mail_activity_board/static/src/components/chatter_topbar/chatter_topbar.js",
+    {
+        // --------------------------------------------------------------------------
+        // Handlers
+        // --------------------------------------------------------------------------
+
+        /**
+         * @privatez
```
Comment (MiquelRForgeFlow) [inline mail_activity_board/static/src/components/chatter_topbar/chatter_topbar.js]:
```suggestion
         * @private
```

**Review by AaronHForgeFlow — APPROVED** [review]
Functional tests :+1:

**Comment by JoanMForgeFlow** [comment]
Functionally, the module works as it should, but the pre-commit fails with the following error:

` 3:1  error  Parsing error: 'import' and 'export' may appear only with 'sourceType: module'`

Anyone have an idea how to solve it?
Thanks in advance
@Tardo 
@MiquelRForgeFlow 
@AaronHForgeFlow

**Comment by AaronHForgeFlow** [comment]
I think this may be related to https://github.com/OCA/oca-addons-repo-template/issues/85 @Yajo

**Comment by AaronHForgeFlow** [comment]
> I think this may be related to [OCA/oca-addons-repo-template#85](https://github.com/OCA/oca-addons-repo-template/issues/85) @Yajo

after some research, it seems the .js files must be renamed to esm.js. So please diregard my previous message.

**Comment by JoanMForgeFlow** [comment]
> > I think this may be related to [OCA/oca-addons-repo-template#85](https://github.com/OCA/oca-addons-repo-template/issues/85) @Yajo
> 
> after some research, it seems the .js files must be renamed to esm.js. So please diregard my previous message.

Solved! Thanks

**Review by AaronHForgeFlow — APPROVED** [review]

**Pull request merged**